Fix part of #10798 Replaced .click() with action.click() from action.js

Prototype4988 opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes or fixes part of #10798.
2. This PR does the following:
Issue #10798 Replaced .click() with action.click() from action.js for core/tests/protractor/learnerFlow.js
## Essential Checklist

- [x] The PR title starts with "Fix #10798 ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #10798: ...".)
- [x] The linter/Karma presubmit checks have passed locally on your machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
  - This lets reviewers restart your CircleCI tests for you.
- [x] The PR is made from a branch that's **not** called "develop".

## Proof that changes are correct

<!--
Add videos/screenshots of the user-facing interface to demonstrate that the changes made in this PR work correctly.
-->

## PR Pointers

- Make sure to follow the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change).
- Oppiabot will notify you when you don't add a PR_CHANGELOG label. If you are unable to do so, please @-mention a code owner (who will be in the Reviewers list), or ask on [Gitter](https://gitter.im/oppia/oppia-chat).
- For what code owners will expect, see the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
- Make sure your PR follows conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays.
- Never force push. If you do, your PR will be closed.
- Oppiabot can assign anyone for review/help if you leave a comment like the following: "{{Question/comment}} @{{reviewer_username}} PTAL"
- Some of the e2e tests are flaky, and can fail for reasons unrelated to your PR. We are working on fixing this, but in the meantime, if you need to restart the tests, please check the ["If your build fails" wiki page](https://github.com/oppia/oppia/wiki/If-your-build-fails).

Prototype4988 committed:
Issue #10798 Replaced .click() with action.click() from action.js for core/tests/protractor/learnerFlow.js

Prototype4988 committed:
#10798 Replaced .click() with action.click() from action.js

Prototype4988 commented:
Hi, Could You check why my pr is failing ci/circleci: e2e_new_structure_tests

kevintab95 commented on core/tests/protractor/learnerFlow.js:
```diff
@@ -50,7 +51,7 @@ describe('Learner dashboard functionality', function() {
   var clickContinueButton = async function() {
     await waitFor.elementToBeClickable(
```
When you do a "action.click" for an element, it internally does a "waitFor.elementToBeClickable" so this line becomes redundant and can be dropped. Can you please remove this line and do the same below in similar cases?

Prototype4988 replied:
Thanks @kevintab95 I updated this changes PTLA

kevintab95 reviewed (changes requested):
Thanks @Prototype4988! Left a comment. PTAL!

U8NWXD reviewed (approved):
Approving since kevintab95 already reviewed the e2e tests

Prototype4988 committed:
Update learnerFlow.js

@kevintab95 Updated the required changes. Kindly review it and let me know if there's any further changes.

Prototype4988 commented:
@kevintab95 Updated the required changes. Kindly review it and let me know if there's any further changes.

sajalasati commented:
> @kevintab95 Updated the required changes. Kindly review it and let me know if there's any further changes.

@Prototype4988 Please also make sure the reviewers are assigned to the PR and you are un-assigned. You can do this by leaving a `PTAL @reviewer` comment (adding all reviewer names in it) so that Oppiabot can do that for you. Also make sure you reply to each of the review comment before requesting for re-review. 
You can read more about both in our [guide](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#address-review-comments-until-all-reviewers-give-lgtm-looks-good-to-me) - make sure you follow this practice in this and other future PRs. Thanks!

Prototype4988 commented:
PTLA @kevintab95 @U8NWXD

U8NWXD commented:
I'm deassigning myself since kevintab95 is handling the e2e test review for this PR (I share code ownership with them)

kevintab95 reviewed (approved):
Thanks @Prototype4988! LGTM

U8NWXD reviewed (approved):
Approving since kevintab95 already reviewed the e2e tests. I also restarted the flaking tests

U8NWXD commented:
@Prototype4988 please merge from develop as Oppiabot [requested](https://github.com/oppia/oppia/pull/12152#issuecomment-799224182)

U8NWXD commented:
@Prototype4988 we've updated the CI checks, and we can't merge your PR until you merge in the updated config files from develop

FareesHussain reviewed (changes requested):
Update the PR title according to the checklist i.e, `Fix part of #10798: Replaced.....`
